src/indicators/drought/config.py

```diff
@@ -16,107 +17,138 @@ class Config:
     ICPAC_LAT = "y"
 
     ### NMME
-    #date should be string of YYYYMM
-    #TODO: in future might want to add support for monthly (and temperature) forecasts
-    NMME_FTP_URL_SEASONAL = "ftp://ftp.cpc.ncep.noaa.gov/NMME/prob//netcdf/prate.{date}.prob.adj.seas.nc"
+    # date should be string of YYYYMM
+    # TODO: in future might want to add support for monthly (and temperature) forecasts
+    NMME_FTP_URL_SEASONAL = (
+        "ftp://ftp.cpc.ncep.noaa.gov/NMME/prob//netcdf/prate.{date}.prob.adj.seas.nc"
+    )
     NMME_DIR = "nmme"
     NMME_NC_FILENAME_RAW = "nmme_prate_{date}_prob_adj_seas.nc"
     NMME_NC_FILENAME_CRS = "nmme_prate_{date}_prob_adj_seas_{tercile}_crs.nc"
     NMME_LOWERTERCILE = "prob_below"
     NMME_LON = "x"
     NMME_LAT = "y"
 
-
     ### CHIRPS
     CHIRPS_DIR = "chirps"
     CHIRPS_MONTHLY_DIR = "monthly"
     CHIRPS_SEASONAL_DIR = "seasonal"
     # resolution can be 25 or 5
-    CHIRPS_FTP_URL_GLOBAL_DAILY="https://data.chc.ucsb.edu/products/CHIRPS-2.0/global_daily/netcdf/p{resolution}/chirps-v2.0.{year}.days_p{resolution}.nc"
-    CHIRPS_FTP_URL_AFRICA_DAILY="https://data.chc.ucsb.edu/products/CHIRPS-2.0/africa_daily/tifs/p{resolution}/{year}/chirps-v2.0.{year}.{month}{day}.tif"
-    CHIRPS_FTP_URL_GLOBAL_MONTHLY="https://data.chc.ucsb.edu/products/CHIRPS-2.0/global_monthly/netcdf/chirps-v2.0.monthly.nc"
+    CHIRPS_FTP_URL_GLOBAL_DAILY = "https://data.chc.ucsb.edu/products/CHIRPS-2.0/global_daily/netcdf/p{resolution}/chirps-v2.0.{year}.days_p{resolution}.nc"
+    CHIRPS_FTP_URL_AFRICA_DAILY = "https://data.chc.ucsb.edu/products/CHIRPS-2.0/africa_daily/tifs/p{resolution}/{year}/chirps-v2.0.{year}.{month}{day}.tif"
+    CHIRPS_FTP_URL_GLOBAL_MONTHLY = "https://data.chc.ucsb.edu/products/CHIRPS-2.0/global_monthly/netcdf/chirps-v2.0.monthly.nc"
     CHIRPS_NC_FILENAME_RAW = "chirps_global_daily_{year}_p{resolution}.nc"
     CHIRPS_NC_FILENAME_CRS = "chirps_global_daily_{year}_p{resolution}_crs.nc"
-    CHIRPS_MONTHLY_RAW_DIR = Path(DATA_DIR) / PUBLIC_DIR / RAW_DIR / GLOBAL_ISO3 / CHIRPS_DIR / CHIRPS_MONTHLY_DIR
-    CHIRPS_MONTHLY_RAW_PATH = CHIRPS_MONTHLY_RAW_DIR / f"chirps_{GLOBAL_ISO3}_monthly.nc"
+    CHIRPS_MONTHLY_RAW_DIR = (
+        Path(DATA_DIR)
+        / PUBLIC_DIR
+        / RAW_DIR
+        / GLOBAL_ISO3
+        / CHIRPS_DIR
+        / CHIRPS_MONTHLY_DIR
+    )
+    CHIRPS_MONTHLY_RAW_PATH = (
+        CHIRPS_MONTHLY_RAW_DIR / f"chirps_{GLOBAL_ISO3}_monthly.nc"
+    )
     CHIRPS_MONTHLY_COUNTRY_FILENAME = "{country_iso3}_chirps_monthly.nc"
-    CHIRPS_SEASONAL_LOWERTERCILE_COUNTRY_FILENAME = "{country_iso3}_chirps_seasonal_lowertercile.nc"
-    CHIRPS_LON = "longitude" #"x" #
-    CHIRPS_LAT = "latitude" #"y" #
+    CHIRPS_SEASONAL_LOWERTERCILE_COUNTRY_FILENAME = (
+        "{country_iso3}_chirps_seasonal_lowertercile.nc"
+    )
+    CHIRPS_LON = "longitude"  # "x" #
+    CHIRPS_LAT = "latitude"  # "y" #
     CHIRPS_VARNAME = "precip"
 
-
+    ### CHIRPS-GEFS
+    CHIRPSGEFS_DIR = "chirpsgefs"
```

Comment:
Should this maybe be "chirps_gefs"? That's the name of the dir in `Data/public/raw/glb/`

Reply:
`chirps_gefs` is the old directory and is deleted on my local version as well as on the GDrive online. Is the dir for you still visible?

Reply:
Ah you are right, that is so weird. I deleted my local version, let me know if this causes you any issues...